src/xpcc/architecture/platform/driver/can/stm32/can.cpp.in

```diff
@@ -35,13 +35,6 @@
 #define CAN_BTR_TS2_POS		20
 #define CAN_BTR_TS1_POS		16
 
-%% if target is stm32f3
-#	define	XPCC_ISR_NAME(CAN1_TX)		XPCC_ISR_NAME(USB_HP_CAN1_TX)
-#	define	XPCC_ISR_NAME(CAN1_RX0)		XPCC_ISR_NAME(USB_LP_CAN1_RX0)
-#	define	CAN1_TX_IRQn				USB_HP_CAN1_TX_IRQn
-#	define	CAN1_RX0_IRQn				USB_LP_CAN1_RX0_IRQn
-%% endif
-
```

Comment:
Note to self:
1. This never worked, because `#define XPCC_ISR_NAME(CAN1_TX)` declared a new macro with `CAN1_TX` as an argument. Hence the warning about redefinition.
2. This is not necessary anymore, because the CMSIS headers for STM32F3 include these redefines.
```cpp
#define CAN_TX_IRQn     USB_HP_CAN_TX_IRQn
#define CAN_TX_IRQHandler     USB_HP_CAN_TX_IRQHandler
#define USB_LP_CAN_RX0_IRQn     CAN_RX0_IRQn
#define USB_LP_CAN_RX0_IRQHandler     CAN_RX0_IRQHandler
```